src/dotnet/Azure.ClientSdk.Analyzers/Azure.ClientSdk.Analyzers/ClientMethodsAnalyzer.cs

```diff
@@ -311,7 +311,22 @@ private static bool DoReturnTypesMatch(ITypeSymbol asyncReturnType, ITypeSymbol
                 else
                 {
                     var asyncInnerType = asyncNamedType.TypeArguments.FirstOrDefault();
+
+                    string asyncInnerTypeName = asyncNamedType.Name;
```

Comment:
```suggestion
                    var asyncInnerTypeName = asyncNamedType.Name;
```

nit:  Everything else around here is `var`.   Embrace the `var`!